src/app/modules/admin-core/components/admin-footer/admin-footer.component.ts

```diff
@@ -10,7 +10,7 @@ export class AdminFooterComponent implements OnInit {
   constructor() {
     this.footerLink = [
         {
-            name: "Copyright @2019 Shikshalokam"
+            name: "Copyright @2020 Shikshalokam"
```

Comment:
what happens for next year?

Reply:
That footer comming from portal library current statically i made same footer

Reply:
I understood that i am telling we should put Date function to put the year